MeNext/Layout/MainPage.cs

```diff
@@ -63,7 +65,7 @@ public void SomethingChanged()
                     foreach (var page in this.pages) {
                         this.Children.Add(page);
                     }
-                } else {
+                } else { // No Event no tabs.
```

Comment:
Obviously don't go crazy with following these patterns, but they should be a rule of thumb